ddtrace/contrib/django/utils.py

```diff
@@ -85,9 +87,23 @@ def get_request_uri(request):
             log.debug("Failed to build Django request host", exc_info=True)
             host = "unknown"
 
+    # If request scheme is missing, possible in case where wsgi.url_scheme
+    # environ has not been set, then default to http
+    scheme = request.scheme or "http"
+
     # Build request url from the information available
     # DEV: We are explicitly omitting query strings since they may contain sensitive information
-    urlparts = dict(scheme=request.scheme, netloc=host, path=request.path, params=None, query=None, fragment=None)
+    urlparts = dict(scheme=scheme, netloc=host, path=request.path, params=None, query=None, fragment=None)
+
+    # If any url part is a SimpleLazyObject, use it's __class__ property to cast
+    # str/bytes and allow for _setup() to execute
+    for (k, v) in urlparts.items():
+        if isinstance(v, SimpleLazyObject):
+            if v.__class__ == str:
```

Comment:
```suggestion
            if v.__class__ is str:
```

should these be `is` checks? Or maybe even `isinstance`? Flashbacks to when we assumed primitive type usage in the encoder